metricFileReplay/README.md

```diff
@@ -96,39 +110,36 @@ ifLicenseKey=
 ifSamplingInterval=1
 
 ##Agent type
+#
 ifAgent=metricFileReplay
+#
 
 ##The server reporting Url(Do not change unless you have on-prem deployment)
 ifReportingUrl=https://app.insightfinder.com
 ```
 
-
-3) Download the agent Code which will be distributed to other machines(not required if you have the offline installation package)
-```
-cd files
-sudo -E ./downloadAgentSSL.sh
-or
-sudo -E ./downloadAgentNoSSL.sh
+5) Run the playbook
 ```
-4) Run the playbook(Go back to the DeployAgent directory)
-```
-cd ..
 ansible-playbook insightagent.yaml
 ```
 
 ### Sending Data
-1) Put data files in /root/InsightAgent-master/data/
-Make sure each file is .csv formatted, starts with a row of headers and the headers should have "timestamp" field in it.
+1) Make sure each file is .csv formatted, starts with a row of headers and the headers should have "timestamp" field in it.
 
 2) Run the following command for each data file.
 ```
-sudo python /root/InsightAgent-master/common/reportMetrics.py -m metricFileReplay -f PATH_TO_CSV_FILE
+sudo python common/reportMetrics.py -w https://app.insightfinder.com -m metricFileReplay -f PATH/TO/CSV_FILE
+```
+Note: If replaying to an on-prem installation, add the server ip and port after the -w option.
+
+If you want to send a list of logs within a directory, you can use:
+```
+find /PATH/TO/DIRECTORY -maxdepth 1 -type f -exec python common/reportMetrics.py... -f {} \;
 ```
-Where PATH_TO_CSVFILENAME is the path and filename of the csv file.
 
 If you are replaying the output of a sar file, you can specify so as an argument to the -t parameter:
 ```
-sudo python common/reportMetrics.py -t sar -m metricFileReplay -f PATH_TO_SAR_FILE
+sudo python common/reportMetrics.py -w https://app.insightfinder.com -m metricFileReplay -t sar -f PATH/TO/SAR_FILE
```

Comment:
Not always app.insightfinder.com add a placeholder host

Reply:
that is the placeholder; it's mentioned above that you may need to change it.